ingress/manifests/base/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ingress-proxy
+  namespace: system
+  labels:
+    app: ingress-proxy
+
+spec:
+  replicas: 1
```

Comment:
One replica only, should we add more for resiliency?

Reply:
that can be increased directly in the stage/prod overlays, for the base (that's also used for tests) I'd suggest to leave it to 1